### DISCUSSION TITLE
ActivityPub support for federation?

### DISCUSSION THREAD

**Discussion by poVoq** [rauversion/rauversion-phx#46] [General]
Might be useful given the social nature of this and also to interact with similar platforms such as Funkwhale.

**Comment by michelson** [comment]
This would be a great addition. We have discussed this with the team. we could use this issue to summarize our findings in this regard.

**Comment by poVoq** [comment]
Cool!

Maybe this helps: https://github.com/bonfire-networks/activity_pub ?

But there are also other libraries in Erlang I think.

**Comment by michelson** [comment]
that library seems the way to go!, also I've checked this article on plemora https://medium.com/we-distribute/activitypub-support-lands-in-pleroma-96635b7d6fde it may be good for reference.

**Comment by poVoq** [comment]
https://framagit.org/framasoft/mobilizon

Would be probably possible to use as a base for federating events.
